joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: $(arg joint_state_frequency)

$(arg prefix)controller:
  type: position_controllers/GripperActionController
  joint: $(arg prefix)finger_joint
  action_monitor_rate: 20
  goal_tolerance: 0.01
  max_effort: 100
  stall_velocity_threshold: 0.01
  stall_timeout: 0.2
  gains:
    $(arg prefix)finger_joint: {p: 200, d: 1, i: 5, i_clamp: 1}
